src/Sarif/Writers/ConsoleLogger.cs

```diff
@@ -90,7 +91,34 @@ public void AnalyzingTarget(IAnalysisContext context)
 
             WriteLineToConsole(string.Format(CultureInfo.CurrentCulture,
                     SdkResources.MSG001_AnalyzingTarget,
-                        context.TargetUri.GetFileName()));
+                        context.TargetUri.LocalPath));
+        }
+
+        public void CompletedAnalyzingTarget(IAnalysisContext context)
+        {
+            if (context == null)
+            {
+                throw new ArgumentNullException(nameof(context));
+            }
+
+            WriteLineToConsole(string.Format(CultureInfo.CurrentCulture,
+                    SdkResources.MSG_CompletedAnalyzingTarget,
+                        context.TargetUri.LocalPath));
+        }
+
+        public void LogMemoryUsage(IAnalysisContext context)
```

Comment:
>LogMemoryUsage [](http://example.com/codeflow?start=20&length=14)

A better pattern would be for the context object to indicate whether memory logging is in play and to emit this data when we either start or stop analyzing a target.

Reply:
Yes, see the other comment.